testament/categories.nim

```diff
@@ -482,6 +482,17 @@ proc makeSupTest(test, options: string, cat: Category): TTest =
   result.options = options
   result.startTime = epochTime()
 
+const maxRetries = 3
+template retryCommand(call): untyped =
+  var res: typeof(call)
+  var backoff = 1
+  for i in 0..<maxRetries:
+    res = call
+    if res.exitCode == QuitSuccess or i == maxRetries: break
```

Comment:
@Clyybber this should be `i == maxRetries-1`

Reply:
```suggestion
    if res.exitCode == QuitSuccess or i == maxRetries-1: break
```